.github/workflows/docs.yml

```diff
@@ -69,4 +69,15 @@ jobs:
         run: npm ci
 
       - name: 🔎 Send to Algolia
-        run: npm run buildsearch
\ No newline at end of file
+        uses: actions/checkout@v2
+
+      - name: Get the content of algolia.json as config
+        id: algolia_config
+        run: echo "::set-output name=config::$(cat algolia.json | jq -r tostring)"
```

Comment:
I think you need to install `jq` first? CMIIW but I don't think Github Runner has JQ by default